electron/src/plugins.ts

```diff
@@ -0,0 +1,18 @@
+import { Plugins } from './global';
+import { mergeWebPlugins, mergeWebPlugin, WebPlugin } from '../../core/src/web/index';
```

Comment:
These should be importing from `@capacitor/core`

Reply:
using `@capacitor/core` is giving build grief